docs/internal/Versioning.md

```diff
@@ -31,78 +31,113 @@ This version is then accessible through the `getTransportVersion` method
 on `StreamInput` and `StreamOutput`, so serialization code can read/write
 objects in a form that will be understood by the other node.
 
-Every change to the transport protocol is represented by a new transport version,
-higher than all previous transport versions, which then becomes the highest version
-recognized by that build of Elasticsearch. The version ids are stored
-as constants in the `TransportVersions` class.
+Internally, every change to the transport protocol is represented by a new transport version, higher than all previous
+transport versions, which then becomes the highest version recognized by that build of Elasticsearch. The version ids
+are stored
+in state files in the `transport/` resources directory.
 Each id has a standard pattern `M_NNN_S_PP`, where:
+
 * `M` is the major version
 * `NNN` is an incrementing id
 * `S` is used in subsidiary repos amending the default transport protocol
 * `PP` is used for patches and backports
```

Comment:
This reference to "backports" now seems a little confusing. Is it still true that this is used for backports?